mysql-test/suite/tokudb.parts/t/partition_mgm.inc

```diff
@@ -0,0 +1,668 @@
+#
```

Comment:
I decided to place it in t directory because of three reasons: 1) the suite does not contain "include" directory; 2) the separate inc file for this test is temporary solution, we will remove it after inplace partition altering implementation; 3) there are a lot of other inc files placed not in "include" subdirectory in the source tree.